pydra/engine/tests/test_specs.py

```diff
@@ -27,7 +27,7 @@
 
 def test_basespec():
     spec = BaseSpec()
-    assert spec.hash == "06fe829a5dca34cc5f0710b454c24808"
+    assert spec.hash == "0b1d98df22ecd1733562711c205abca2"
```

Comment:
Yes, it doesn't quite feel right to hard code this. However, it does flag whether the hash is stable across versions (or isn't in this case). Ideally it would be so that caches wouldn't be invalidated by new Pydra versions, but in this case, I'm not sure whether there is an easy way around this

Reply:
I believe I added these kind of tests to not change the hashing way without noticing